sources/@roots/bud-preset-wordpress/config/stylelint.cjs

```diff
@@ -3,6 +3,11 @@
  */
 module.exports = {
   rules: {
-    'custom-property-pattern': null,
+    'custom-property-pattern': [
+      '^([a-z][a-z0-9]*)(-{1,2}[a-z0-9]+)*$',
```

Comment:
this is passing eslint cos it's a cjs file but can you quote this string with backticks. thanks!

Reply:
I'm sorry, I don't understand your comment. Isn't this value already a string literal?

Reply:
instead of `'` quotes use `